### ISSUE TITLE
pruned_weights

### ISSUE THREAD

**Issue by PeterBishop0** [FarinaMatteo/multiflow#1]
Thanks for your great work! @FarinaMatteo 
Can you provide the off-the-shelf pruned_weights and fine-tuned weights for vqa?

**Comment by FarinaMatteo** [comment]
Hi @PeterBishop0! First of all, thanks for having a look at our work 😊 Sorry it took me a week to get back to you, but I had some days off.

You can access all the pruning masks of the pretrained models [here](https://drive.google.com/drive/folders/11DDZsEFV86YsXHFvLqIXHL_prwsNaqpg?usp=drive_link) for all the models we tested (BLIP, XVLM and DINO). For each model you will find 9 pruning masks, spanning the combinations of 3 seeds (0,1,6996) and 3 sparsity levels (63%, 75%, and 90%, as per the experiments of the paper). I will add this to the README of the repo.

Please note that these are the pruning masks only, and you still need to fine-tune your pruned models. The code has undergone major refactoring before the release, so I am still spawning jobs to ensure full reproducibility (so far everything is reproducible, and I totally do not expect any problem, but I want to make sure of it). 

I am planning to also upload all checkpoints for all tasks, so I'll get back to you when I'm done in case you still need the VQA checkpoints.

**Comment by PeterBishop0** [comment]
Thanks a lot!
